runners/google-cloud-dataflow-java/pom.xml

```diff
@@ -133,15 +133,15 @@
                     <!-- com.google.common is too generic, need to exclude guava-testlib -->
                     <exclude>com.google.common.**.testing.*</exclude>
                   </excludes>
-                  <shadedPattern>org.apache.beam.sdk.repackaged.com.google.common</shadedPattern>
+                  <shadedPattern>org.apache.beam.runners.dataflow.repackaged.com.google.common</shadedPattern>
```

Comment:
I assumed we'd just match the root Java package of the module (`org.apache.beam.runners.dataflow`, in this case). Thoughts?